substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/SecurityServicesFeature.java

```diff
@@ -109,6 +109,11 @@ public void duringSetup(DuringSetupAccess access) {
         ImageSingletons.lookup(RuntimeClassInitializationSupport.class).rerunInitialization(clazz(access, "sun.security.provider.SeedGenerator"), "for substitutions");
         ImageSingletons.lookup(RuntimeClassInitializationSupport.class).rerunInitialization(clazz(access, "sun.security.provider.SecureRandom$SeederHolder"), "for substitutions");
 
+        if(JNIRegistrationUtil.isWindows() && JavaVersionUtil.JAVA_SPEC >= 11) {
```

Comment:
Since `sun.security.provider.AbstractDrbg$SeederHolder` is a shared class this should be enabled for Linux/Darwin platforms too, not only Windows.

Reply:
@cstancu Thank you for the comment. I will.

As I am learning this, could you take a minute to explain why the bug does not manifest on Linux for me, please? The class is used even with my Linux native-image build:

```
$ grep AbstractDrbg /home/karm/workspaceRH/UUID/reports/build_time_classes_20200422_090211.txt
sun.security.provider.AbstractDrbg
sun.security.provider.AbstractDrbg$$Lambda$674/0x00000007c0bd8c40
sun.security.provider.AbstractDrbg$NonceProvider
sun.security.provider.AbstractDrbg$SeederHolder
sun.security.provider.AbstractDrbg$SeederHolder$$Lambda$680/0x00000007c0bdf440
```

Although it is apparently not necessary to re-initialize it at runtime to get a new seed on startup, see:

```
$ strace ./main 

access("/dev/random", R_OK)             = 0
access("/dev/random", R_OK)             = 0
access("/dev/urandom", R_OK)            = 0
lstat("/dev", {st_mode=S_IFDIR|0755, st_size=4200, ...}) = 0
lstat("/dev/random", {st_mode=S_IFCHR|0666, st_rdev=makedev(1, 8), ...}) = 0
open("/dev/random", O_RDONLY)           = 3
fstat(3, {st_mode=S_IFCHR|0666, st_rdev=makedev(1, 8), ...}) = 0
lstat("/dev", {st_mode=S_IFDIR|0755, st_size=4200, ...}) = 0
lstat("/dev/urandom", {st_mode=S_IFCHR|0666, st_rdev=makedev(1, 9), ...}) = 0
open("/dev/urandom", O_RDONLY)          = 4
fstat(4, {st_mode=S_IFCHR|0666, st_rdev=makedev(1, 9), ...}) = 0
read(4, "\327?\206LS\r\235x\332\3723b|tv'\304\202=\254", 20) = 20
read(4, "^\310\333 \37\"\1\0323\23\34\346U\224S\374\23\354?\235%]\2\213\356P\26\355\351l\207l"..., 256) = 256
```

Thank you for your time.

K.